jib-core/src/test/java/com/google/cloud/tools/jib/http/RequestWrapper.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.http;
+
+/** Helper to expose package-private methods. */
```

Comment:
Any reason we can't just make the method public and mark with `@VisibleForTesting`?

Reply:
I often don't like adding a new method to production code that will be used only by testing, though it depends.

Reply:
WDYT? Although this case doesn't add a new method, making a method `public` has a significant implication likewise.

Reply:
Hmm, I guess it's fine. Can always clean up later if we decide differently.

Reply:
Okay, at least this class replaces the more complex `MockConnection`, so it doesn't make the situation worse.